src/gui/optionsdlg.cpp

```diff
@@ -72,6 +74,23 @@
 
 #include "ui_optionsdlg.h"
 
+class WheelEventEater : public QObject
+{
+public:
+    using QObject::QObject;
+
+private:
+    bool eventFilter(QObject *obj, QEvent *event) override
+    {
+        switch (event->type()) {
```

Comment:
Why did you write 5 lines instead of a single-line return statement?

Reply:
because your suggestion just bumped out now :)

Reply:
PArdon me, thought that's kinda obvious thing and you had a motivation to write as it is.